src/main/java/io/github/apickledwalrus/skriptplaceholders/skript/elements/ExprPlaceholderResult.java

```diff
@@ -56,6 +57,7 @@ protected String[] get(Event event) {
 	public Class<?>[] acceptChange(ChangeMode mode) {
 		switch (mode) {
 			case SET:
+				return CollectionUtils.array(Object.class);
 			case DELETE:
 			case RESET:
 				return CollectionUtils.array(String.class);
```

Comment:
```suggestion
			case SET:
			case DELETE:
			case RESET:
				return CollectionUtils.array(Object.class);
```
you can just change it here :) the contents of the array do not matter for DELETE/RESET